frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -402,6 +402,18 @@ export default class Runs extends Vue {
     });
   }
 
+  get sortedRuns(): types.Run[] {
+    return this.runs.slice().sort((a, b) => {
+      if (a.time < b.time) {
+        return 1;
+      }
+      if (a.time > b.time) {
+        return -1;
+      }
+      return 0;
+    });
```

Comment:
```suggestion
    return this.runs.slice().sort((a, b) => a.time - b.time);
```

esta es la manera idiomática de hacerlo: https://github.com/Microsoft/TypeScript/issues/24451#issuecomment-392735550